docs/source/getting_started/changelog.md

```diff
@@ -0,0 +1,18 @@
+# Changelog
+
+## v1.0.0rc0 (08/07/2020)
+
+- Rename python-runner to python-editor [#721](https://github.com/elyra-ai/elyra/issues/721)
+- Add support for JupyterLab Dark theme [#706](https://github.com/elyra-ai/elyra/issues/706)
+- Improve dependency management and archive generation [#702](https://github.com/elyra-ai/elyra/issues/702)
+- Properly set pipeline version on new pipelines [#698](https://github.com/elyra-ai/elyra/issues/698)
+- Make pipeline process and export asynchronous [#695](https://github.com/elyra-ai/elyra/issues/695)
+- Use absolute form of export path when exporting pipelines [#690](https://github.com/elyra-ai/elyra/issues/690)
+- Validate pipeline and export submission dialogs [#684](https://github.com/elyra-ai/elyra/issues/684)
+- Create metadata editor [#589](https://github.com/elyra-ai/elyra/issues/589)
+
+Note: Pipeline nodes that currently do not list any dependencies but have 'Include Subdirectories' enabled, 
+will not have the intended result. Instead, those node properties must be updated to include '*' in the dependencies list. 
+However, use of '*' with 'Include Subdirectories' is not recommended and finer grained dependency lists should be utilized.
```

Comment:
```suggestion
However, use of '\*' with 'Include Subdirectories' is not recommended and finer grained dependency lists should be utilized.
```

Reply:
^^ The `*` isn't rendered properly in the markdown. Not sure why escaping is required.